modules/Hoot/managers/api.js

```diff
@@ -1227,22 +1225,23 @@ export default class API {
             } );
     }
 
-    differentialStats( jobId, includeTags ) {
+    changesetStats( jobId, includeTags ) {
```

Comment:
renamed a bunch of method from "differential" to "changeset" to reflect that it's not only for the differential changeset workflow